Profiles/mp-MedicationAdministration2.xml

```diff
@@ -1,568 +1,569 @@
+    <name value="ART-DECOR Dataset MP9 3.0.0-beta.1 20230217" />
+  </mapping>
+  <mapping>
+    <identity value="mp-dataset-mp9-300-beta2-20231017" />
+    <uri value="https://decor.nictiz.nl/pub/medicatieproces/mp-html-20231017T223005/ds-2.16.840.1.113883.2.4.3.11.60.20.77.1.4-2022-06-30T000000.html" />
+    <name value="ART-DECOR Dataset MP9 3.0.0-beta.2 20231017" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="MedicationAdministration" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/MedicationAdministration" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="MedicationAdministration">
+      <path value="MedicationAdministration" />
+      <short value="MedicationAdministration" />
+      <alias value="Medicatietoediening" />
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.20928" />
+        <comment value="MedicationAdministration" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.extension:agreedDateTime">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="agreedDateTime" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationAdministration2.AgreedDateTime" />
+      </type>
+    </element>
+    <element id="MedicationAdministration.extension:agreedDateTime.value[x]">
+      <path value="MedicationAdministration.extension.value[x]" />
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23171" />
+        <comment value="AgreedDateTime" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.extension:asAgreedIndicator">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="asAgreedIndicator" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-AsAgreedIndicator" />
+      </type>
+    </element>
+    <element id="MedicationAdministration.extension:asAgreedIndicator.value[x]">
+      <path value="MedicationAdministration.extension.value[x]" />
+      <mapping>
+        <identity value="mp-dataset-mp9-300-beta1-20230217" />
+        <map value="mp-dataelement9x-429" />
+        <comment value="AsAgreedIndicator" />
+      </mapping>
+      <mapping>
+        <identity value="mp-dataset-mp9-300-beta2-20231017" />
+        <map value="mp-dataelement9x-429" />
+        <comment value="AsAgreedIndicator" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.extension:medicationAdministrationReasonForDeviation">
+      <path value="MedicationAdministration.extension" />
+      <sliceName value="medicationAdministrationReasonForDeviation" />
+      <max value="1" />

```

Comment:
It's better to change this cardinality in the extension itself. Compare this with for example the InjectionPatchSite extension present in the mp-MedicationAdministration2 profile.

Reply:
After changing the cardinality in the extension and saving, reloading the mp-MedicationAdministration2 profile in Forge should automatically remove the max cardinality of 1 in that profile.

Reply:
Oops, only read the first comment. Reverted the previous commit and changed the cardinality in the extension.